Fix extra semicolon warnings

Following error message:
```

/usr/local/include/pcl-1.13/pcl/filters/voxel_grid.h:193:36: error: extra ‘;’ [-Werror=pedantic]
  193 |       PCL_MAKE_ALIGNED_OPERATOR_NEW;
```

Problem is the extra semicolon here:
https://github.com/PointCloudLibrary/pcl/blob/30124ee1ab5518a5cc7d03a186c46caf9aa99513/filters/include/pcl/filters/voxel_grid.h#L193

Solution is to remove that semicolon, as also seen here:
https://github.com/PointCloudLibrary/pcl/blob/30124ee1ab5518a5cc7d03a186c46caf9aa99513/filters/include/pcl/filters/crop_box.h#L62

--- a/filters/include/pcl/filters/plane_clipper3D.h
+++ b/filters/include/pcl/filters/plane_clipper3D.h
@@ -54,7 +54,7 @@ namespace pcl
       using Ptr = shared_ptr< PlaneClipper3D<PointT> >;
       using ConstPtr = shared_ptr< const PlaneClipper3D<PointT> >;
 
-      PCL_MAKE_ALIGNED_OPERATOR_NEW;
+      PCL_MAKE_ALIGNED_OPERATOR_NEW
 
       /**
        * @author Suat Gedikli <gedikli@willowgarage.com>

--- a/filters/include/pcl/filters/uniform_sampling.h
+++ b/filters/include/pcl/filters/uniform_sampling.h
@@ -71,7 +71,7 @@ namespace pcl
       using Ptr = shared_ptr<UniformSampling<PointT> >;
       using ConstPtr = shared_ptr<const UniformSampling<PointT> >;
 
-      PCL_MAKE_ALIGNED_OPERATOR_NEW;
+      PCL_MAKE_ALIGNED_OPERATOR_NEW
 
       /** \brief Empty constructor. */
       UniformSampling (bool extract_removed_indices = false) :

--- a/filters/include/pcl/filters/voxel_grid.h
+++ b/filters/include/pcl/filters/voxel_grid.h
@@ -190,7 +190,7 @@ namespace pcl
       using Ptr = shared_ptr<VoxelGrid<PointT> >;
       using ConstPtr = shared_ptr<const VoxelGrid<PointT> >;
 
-      PCL_MAKE_ALIGNED_OPERATOR_NEW;
+      PCL_MAKE_ALIGNED_OPERATOR_NEW
 
       /** \brief Empty constructor. */
       VoxelGrid () :

--- a/filters/include/pcl/filters/voxel_grid_occlusion_estimation.h
+++ b/filters/include/pcl/filters/voxel_grid_occlusion_estimation.h
@@ -67,7 +67,7 @@ namespace pcl
 
     public:
 
-      PCL_MAKE_ALIGNED_OPERATOR_NEW;
+      PCL_MAKE_ALIGNED_OPERATOR_NEW
 
       /** \brief Empty constructor. */
       VoxelGridOcclusionEstimation ()

--- a/gpu/kinfu_large_scale/include/pcl/gpu/kinfu_large_scale/point_intensity.h
+++ b/gpu/kinfu_large_scale/include/pcl/gpu/kinfu_large_scale/point_intensity.h
@@ -47,7 +47,7 @@
 struct EIGEN_ALIGN16 PointIntensity
 {
 
-  PCL_MAKE_ALIGNED_OPERATOR_NEW;
+  PCL_MAKE_ALIGNED_OPERATOR_NEW
   union
   {
     struct

--- a/registration/include/pcl/registration/gicp.h
+++ b/registration/include/pcl/registration/gicp.h
@@ -113,7 +113,7 @@ public:
       typename IterativeClosestPoint<PointSource, PointTarget, Scalar>::Matrix4;
   using AngleAxis = typename Eigen::AngleAxis<Scalar>;
 
-  PCL_MAKE_ALIGNED_OPERATOR_NEW;
+  PCL_MAKE_ALIGNED_OPERATOR_NEW
 
   /** \brief Empty constructor. */
   GeneralizedIterativeClosestPoint()

--- a/registration/include/pcl/registration/incremental_registration.h
+++ b/registration/include/pcl/registration/incremental_registration.h
@@ -111,7 +111,7 @@ public:
   /** \brief Set registration instance used to align clouds */
   inline void setRegistration(RegistrationPtr);
 
-  PCL_MAKE_ALIGNED_OPERATOR_NEW;
+  PCL_MAKE_ALIGNED_OPERATOR_NEW
 
 protected:
   /** \brief last registered point cloud */

--- a/registration/include/pcl/registration/vertex_estimates.h
+++ b/registration/include/pcl/registration/vertex_estimates.h
@@ -65,7 +65,7 @@ struct PoseEstimate {
   : pose(p), cloud(c)
   {}
 
-  PCL_MAKE_ALIGNED_OPERATOR_NEW;
+  PCL_MAKE_ALIGNED_OPERATOR_NEW
 };
 } // namespace registration
 } // namespace pcl

--- a/test/common/test_transforms.cpp
+++ b/test/common/test_transforms.cpp
@@ -105,7 +105,7 @@ class Transforms : public ::testing::Test
   // Indices, every second point
   Indices indices;
 
-  PCL_MAKE_ALIGNED_OPERATOR_NEW;
+  PCL_MAKE_ALIGNED_OPERATOR_NEW
 };
 
 TYPED_TEST_SUITE (Transforms, TransformTypes);

